xfel/merging/application/phil/phil.py

```diff
@@ -735,11 +735,14 @@
 }
 """
 
-diffbragg_phil = """
-diffBragg {
-  include scope simtbx.diffBragg.phil.phil_scope
-}
-"""
+try:
```

Comment:
Unless I'm missing something this change should have no effect.  All this code does is define the string diffbragg_phil; it will never generate an importerror because the import is done at a later time when the includes are evaluated.